website/content/docs/commands/volume/status.mdx

```diff
@@ -57,15 +58,21 @@ namespace.
 List of all volumes:
 
 ```shell-session
-$ nomad volume [-type csi] status
+$ nomad volume status
+Dynamic Host Volumes
+ID        Name             Namespace  Plugin ID            Node ID   Node Pool  State
+070dc8ca  internal-plugin  default    mkdir                c3c27514  default    ready
+7ce38017  external-plugin  default    example-plugin-mkfs  c3c27514  default    ready
+
+Container Storage Interface
 ID            Name      Namespace  Plugin ID  Schedulable  Access Mode
 ebs_prod_db1  database  default    ebs-prod   true         single-node-writer
 ```
 
-List of all volumes, with external provider info:
+List of all CSI volumes, with external provider info:
 
 ```shell-session
-$ nomad volume [-type csi] status -verbose
+$ nomad volume -type csi status -verbose
```

Comment:
the two bottom examples below here also have this `[]` business - mind cleaning those up while you're in here?